sentry_sdk/_compat.py

```diff
@@ -59,12 +59,12 @@ def reraise(tp, value, tb=None):
 
 def with_metaclass(meta, *bases):
     # type: (Any, *Any) -> Any
-    class metaclass(type):
```

Comment:
I might not be remembering well, but I thought `class metaclass` was common practice?

Reply:
I honestly don't know. Are you saying that metaclasses are supposed to be snakecase?